app/commands/todo/add.rb

```diff
@@ -18,7 +22,8 @@ def initialize(params)
             @aggregate_id = payload.delete(:todo_id)
           end
 
-          def valid?
+          def validate
+            raise BadRequest, 'todo_id is blank' if aggregate_id.nil?
```

Comment:
Yeah, it won't. I still think it makes sense as a validation because it shows what a well formed command looks like.